docs/source/index.rst

```diff
@@ -32,49 +31,54 @@ You can easily deploy Sycamore locally or on a virtual machine using Docker.
 
 With Docker installed:
 
-
-
-1. Clone the Sycamore repo:
+1.	Clone the Sycamore repo: 
 
 ``git clone https://github.com/aryn-ai/sycamore``
 
-2. Set OpenAI Key:
+2.	Set OpenAI Key:
 
 ``export OPENAI_API_KEY=YOUR-KEY``
 
-3. Go to:
+3.	Go to: 
 
 ``/sycamore``
 
-4. Launch Sycamore. Containers will be pulled from DockerHub:
+4.	Launch Sycamore. Conatainers will be pulled from DockerHub:
 
-``docker compose up --pull=always``
+``Docker compose up --pull=always``
 
+5.	The Sycamore demo query UI will be at localhost:3000
 
-**Use the service:**
+You can next choose to run a demo that `prepares and ingests data from the Sort Benchmark website<docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset>`_, [crawl data from a public website](docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website), or write your own data preparation script.
 
-a. Load demo dataset via (NEED LINK tutorial here):
+For more info about Sycamore’s data ingestion and preparation feature set, visit the [Sycamore documentation](docs/source/data_ingestion_and_preparation/data_preparation_concepts.md).
+
+
+Run a demo
+--------------------
+
+a. Load demo dataset using the HTTP crawler (visit tutorial)[docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset]:
 
 ``docker compose run crawl_sort_benchmark``
 
-b. Load custom data via (NEED LINK tutorial here):
+b. Load website data via HTTP crawler (visit tutorial)[docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website]:
```

Comment:
I think this will read a it weird. I would write it as:

(Load website data via HTTP crawler)[docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website]:

Or change the \(\) bit to be "as shown in the tutorial"

Reply:
Ack.